tests/tests_pytorch/accelerators/test_ipu.py

```diff
@@ -319,16 +316,6 @@ def on_predict_batch_end(self, trainer, pl_module, outputs, batch, batch_idx, da
     trainer.predict(model, model.test_dataloader())
 
 
-@RunIf(ipu=True)
```

Comment:
The tests for unsupported accelerators and strategies has been moved to the callback tests below. No need to run on the actual accelerator or launch expensive processes to test this.